.git-blame-ignore-revs

```diff
@@ -1,4 +1,6 @@
 # .git-blame-ignore-revs
 # https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view
 # Switched to clang-format and LLVM style
-9a4dc58421e8bb5042ca0b6aaef711a126c8248b
\ No newline at end of file
+9a4dc58421e8bb5042ca0b6aaef711a126c8248b
+# Adding maybe_unused attributes
+1162ca5f5383eb71ac03a02f9f010471936ff466
```

Comment:
This should only be used for style changes only.

Reply:
OK, I will remove this commit.